lib/beaker/host.rb

```diff
@@ -212,8 +212,13 @@ def ip
     #@return [Boolean] true if x86_64, false otherwise
     def determine_if_x86_64
       if is_cygwin?
-        result = exec(Beaker::Command.new("arch | grep x86_64"), :acceptable_exit_codes => (0...127))
-        result.exit_code == 0
+        if self[:platform] =~ /osx|solaris/
```

Comment:
is_cygwin identified the box as being not-powershell windows - anything that is not in the bucked 'powershell' is identified as not being cygwin.